dlp/snippets/Jobs/list_jobs.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Sample app to list and delete DLP jobs using the Data Loss Prevent API. """
+
+from __future__ import annotations
+
+import argparse
+
+from typing import Optional
+
+import google.cloud.dlp
+
+
+# [START dlp_list_jobs]
```

Comment:
nit: please make sure required imports are inside the region_tag (`google.cloud.dlp` in this case)